misc/services/life_mngr/life_mngr_win.c

```diff
@@ -179,6 +179,7 @@ int main()
 	snprintf(buf, sizeof(buf), SYNC_FMT, WIN_VM_NAME);
 	enable_uart_resend(buf, MIN_RESEND_TIME);
 	send_message_by_uart(hCom2, buf, strlen(buf));
+	Sleep(5U * MS_TO_SECOND);
```

Comment:
Please add some comments.

Reply:
Thanks, have updated this, please double check it.